database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,23 @@
+SET ROLE zalando_nakadi_data_owner;
+
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
```

Comment:
@bocytko If I will rename columns in a way you propose I won't be able to use handy keyword `USING`. I know, that for old zalando db it is important to use the approach you say, but is it mandatory to use exactly the same approach among all microservices?